patroni/postgresql/postmaster.py

```diff
@@ -105,6 +105,28 @@ def from_pid(cls, pid):
         except psutil.NoSuchProcess:
             return None
 
+    def signal_kill(self):
+        """to suspend and kill postmaster and all children
+
+        :returns True if postmaster and children are killed, False if error
+        """
+        try:
+            self.suspend()
+            children = self.children()
+            self.kill()
+        except psutil.NoSuchProcess:
+            pass
+        except psutil.AccessDenied as e:
+            logger.warning("Could not kill PostgreSQL (error: {0})".format(e))
+            return False
+        for child in children:
+            try:
+                child.kill()
+            except psutil.NoSuchProcess:
+                continue
+
```

Comment:
```suggestion
        _, _ = psutil.wait_procs(children.append(self))
```

Reply:
1. The `list.append()` method doesn't return anything
2. We can discards results of `psutil.wait_procs()`